specification.markdown

```diff
@@ -372,7 +387,8 @@ Element Name:   EBMLVersion
     Range:          -
     Default:        1
     Element Type:   Unsigned Integer
-    Description:    The version of EBML parser used to create the file.
+    Description:    The version of EBML parser used to create the EBML
+                    Document.
```

Comment:
Note that these changes from file to EBML Document change the meaning of the spec, but hopefully make it more clear. If a file is an EBML Stream and contains many EBML Documents and thus many EBML Headers, the prior language implied that the EBML Header elements defined the whole file. In the case where the file is an EBML Stream with multiple EBML Headers this led to a contradictory interpretation. I think the references to the EBML Header refering to the EBML Document are more correct than reference the EBML Header to the file.